altos-rust/port/cortex-m0/src/peripheral/usart/control.rs

```diff
@@ -164,8 +296,87 @@ impl CR1 {
         }
     }
 
-    // Sets parity to even or odd.
-    fn set_parity(&self, parity: Parity) {
+    /* Uses bit 5 in CR1 to enable or disable RXNE interrupt based on bool
+     * variable passed in.
+     *      true: Enables interrupt
+     *      false: Disables interrupt
+     *
+     *  Bit 5 RXNEIE: RXNE interrupt enable
+     *      This bit is set and cleared by software.
```

Comment:
Saying it's set and cleared by software seems redundant as a comment on the software setting and clearing the bit... meh